src/Changelog/Query/Requests/Filters/getLogsOfCourseFilter.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace ILIAS\Changelog\Query\Requests\Filters;
+
+
+use ilDateTime;
+
+/**
+ * Class getLogsOfCourseFilter
+ * @package ILIAS\Changelog\Query\Requests\Filters
+ *
+ * @author Theodor Truffer <tt@studer-raimann.ch>
+ */
+class getLogsOfCourseFilter {
+	/**
+	 * @var ilDateTime
+	 */
+	protected $date_from = null;
+	/**
+	 * @var ilDateTime
+	 */
+	protected $date_to = null;
+	/**
+	 * @var int
+	 */
+	protected $event_type = null;
+	/**
+	 * @var int
+	 */
+	protected $user_id = 0;
+
```

Comment:
I just added some standard filters. These may be extended, depending on the usages in the GUI.